src/aioswitcher/api/__init__.py

```diff
@@ -749,9 +843,24 @@ async def set_position(self, position: int = 0) -> SwitcherBaseResponse:
             "logged in session_id=%s, timestamp=%s", login_resp.session_id, timestamp
         )
 
-        packet = packets.RUNNER_SET_POSITION.format(
-            login_resp.session_id, timestamp, self._device_id, hex_pos
-        )
+        if (
+            self._device_type
+            and self.is_using_token
+            and (self._device_type == DeviceType.RUNNER_S11)
+        ):
+            hex_pos = f"0{index}{hex_pos}" if index else hex_pos
+            precommand = "3701"
```

Comment:
It would be better if this will use a similar format we use for other commands.
https://github.com/TomerFi/aioswitcher/blob/dev/src/aioswitcher/api/packets.py

Reply:
@thecode I thought about that but I saw that this general packet is working for both set position, stop position, and also for the lights like turn on and off..
Isn't that better to have 1 single packet for token-based devices instead of duplicating the same packets over and over?

Reply:
You can still move the `precommand ` and a constant to `packets.py` (remember to make it uppercase) and add description there where it is used

Reply:
precommand and command are changing between functions.
for stop_shutter `precommand` is "3702"
for set_position `precommand` is "3701"
for set_light `precommand` is "370a"

`command` is also changing based on "index"
"index" = id that wanted to control
For example, for Runner S11 you have 1 "Runner" and 2 "Lights", So you can control:
Runner: index 1 (which is the default)
Light: index 1 and index 2

For Runner S12 you have 1 "Light" and 2 "Runner", So you can control:
Runner: index 1 and index 2
Light: index 1 (which is the default)

Reply:
Than you need 3 constants:
`STOP_SHUTTER_PRECOMMAND`
`SET_POSITION_PRECOMMAND`
`SET_LIGHT_PRECOMMAND`
we shouldn't have any magic numbers in the code

Reply:
Done. You can resolve it.